src/modules/paymentProcessor/bridging/abstracts/CrosschainBase_v1.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+import {IOrchestrator_v1} from
+    "src/orchestrator/interfaces/IOrchestrator_v1.sol";
+import {IPaymentProcessor_v1} from "@pp/IPaymentProcessor_v1.sol";
+import {IERC20PaymentClientBase_v1} from
+    "@lm/interfaces/IERC20PaymentClientBase_v1.sol";
+import {Module_v1} from "src/modules/base/Module_v1.sol";
+//import {IPP_CrossChain_v1} from "./IPP_Template_v1.sol";
+import {ERC165Upgradeable, Module_v1} from "src/modules/base/Module_v1.sol";
+import {ICrossChainBase_v1} from "./ICrosschainBase_v1.sol";
+/**
+ * @title   Inverter Template Payment Processor
+ *
+ * @notice  Basic template payment processor used as base for developing new payment processors.
+ *
+ * @dev     This contract is used to showcase a basic setup for a payment processor. The contract showcases the
+ *          following:
+ *          - Inherit from the Module_v1 contract to enable interaction with the Inverter workflow.
+ *          - Use of the IPaymentProcessor_v1 interface to facilitate interaction with a payment client.
+ *          - Implement custom interface which has all the public facing functions, errors, events and structs.
+ *          - Pre-defined layout for all contract functions, modifiers, state variables etc.
+ *          - Use of the ERC165Upgradeable contract to check for interface support.
+ *
+ * @custom:security-contact security@inverter.network
+ *                          In case of any concerns or findings, please refer to our Security Policy
+ *                          at security.inverter.network or email us directly!
+ *
+ * @author  Inverter Network
+ */
+
+contract CrosschainBase_v1 is ICrossChainBase_v1, Module_v1 {
```

Comment:
Would assume this to be an abstract contract, as we don't want this contract to work as a standalone

Reply:
Yes this is correct, we changed this in our newest PR but will also adjust here.